test/test_package.py

```diff
@@ -261,6 +261,18 @@ def test_validate_package(self):
             self.assertRaises(InvalidPackage, Package.validate, pack)
             dep_type.remove(depend)
 
+    def test_invalid_package_exception(self):
+        try:
+            raise InvalidPackage('foo')
+        except InvalidPackage as e:
+            self.assertEqual('foo', str(e))
+            self.assertEqual(None, e.package_path)
+        try:
+            raise InvalidPackage('foo', package_path='\\bar')
```

Comment:
Rather then using an escaped backslash I would suggest to using simpler like `/some/path` since it doesn't even have to exist.

Reply:
Fixed.